test/test_helper.rb

```diff
@@ -0,0 +1,5 @@
+# I think we don't need this anymore because Rake automatically adds lib/
+# to $LOAD_PATH before running the tests
+# $LOAD_PATH.unshift File.expand_path("../", __dir__)
```

Comment:
@tomstuart My tests pass with this commented out, and [the docs](https://ruby.github.io/rake/Rake/TestTask.html) say
> libs[RW]
List of directories added to $LOAD_PATH before running the tests. (default is 'lib')

So I'm assuming I don't need to explicitly add this to `$LOAD_PATH` anymore?

Reply:
As discussed during pairing, this is useful for running an individual test file outside of Rake with [`ruby -Itest test/foo_test.rb`](https://github.com/seattlerb/minitest#label-Running+Your+Tests) (rather than [`rake test TEST=test/foo_test.rb`](https://github.com/ruby/rake/blob/v13.0.6/lib/rake/testtask.rb#L31)). So it’s not really necessary, but it’s [normal](https://github.com/rubygems/rubygems/blob/bundler-v2.2.27/bundler/lib/bundler/templates/newgem/test/minitest/test_helper.rb.tt#L3), so we decided to do it for the sake of supporting the `ruby -Itest` approach.